app/src/main/java/com/bignerdranch/android/androidacademy/FragmentMoviesList.kt

```diff
@@ -8,42 +8,47 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import androidx.recyclerview.widget.StaggeredGridLayoutManager
+import kotlinx.coroutines.*
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.Dispatchers.IO
 
-class FragmentMoviesList: Fragment(R.layout.fragment_movies_list), OnItemClickListener {
+class FragmentMoviesList:
+        Fragment(R.layout.fragment_movies_list),
+        OnItemClickListener {
 
     var listener: OnMovieItemClickListener? = null
+    lateinit var movie: List<Movie>
 
     override fun onAttach(context: Context) {
         super.onAttach(context)
         if (context is OnMovieItemClickListener){
             listener = context
         }
+        runBlocking {
```

Comment:
@TimurIsroilov123 you don't need load movies with blocking call
let's redesign this moment